spyder/plugins/toolbar/api.py

```diff
@@ -15,6 +15,7 @@ class ApplicationToolbars:
     File = 'file_toolbar'
     Run = 'run_toolbar'
     Debug = 'debug_toolbar'
+    ControlDebugger = "control_debugger_toolbar"
```

Comment:
Should this use single quotes to follow the surrounding code style?:

```suggestion
    ControlDebugger = 'control_debugger_toolbar'
```